### PR TITLE
Create national-medical-journal-of-india.csl

### PR DESCRIPTION
**Pull request by damnation333**
as it has been discussed here and the code provided by LiborA: https://forums.zotero.org/discussion/59897/style-request-nmji/
Made some minor changes to fit the regulations.

### CODE REVIEW

File: national-medical-journal-of-india.csl [hunk 1]
```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>The National Medical Journal of India</title>
+    <id>http://www.zotero.org/styles/national-medical-journal-of-india</id>
```
Comment (rmzelle) [inline national-medical-journal-of-india.csl]: @damnation333, could you add "the-" to the file name, and style ID and "self" link.

(when in doubt whether "The" is part of a journal's title, I usually check the cover image of the journal, e.g. at http://www.nmji.in/)

Comment (damnation333) [inline national-medical-journal-of-india.csl]: great tip. I'll keep an eye on it from now. :)

**Comment by rmzelle** [comment]
Great, thanks.

**Comment by rmzelle** [comment]
(since you're such a prolific contributor, you might also enjoy http://pinux.info/csls_counter/, which shows the growth of the repository since we moved it to GitHub)

**Pull request merged**